evutil.c

```diff
@@ -372,6 +372,13 @@ evutil_make_listen_socket_reuseable(evutil_socket_t sock)
 }
 
 int
+evutil_socket_bind_address(evutil_socket_t sock, const struct sockaddr *sa,
+    int socklen)
+{
+	return bind(sock, sa, socklen);
```

Comment:
Well, if the library ever gets ported to something where bind() does something different or needs some special setup on particular platforms, having a wrapper already in place would mean isolating the special handling and platform dependencies...

Plus if we wanted to add a tracing message (via `event_debug(())`) every time a socket did a `bind()`, we'd only need to do it in one place...

Did you want _all_ of the invocations of `bind()` addressed?  Including `samples/` and `test/`?